api/src/main/java/org/openmrs/module/pihcore/apploader/CustomAppLoaderFactory.java

```diff
@@ -3804,7 +3804,7 @@ private void configureBasicProgramSummaryApps(String programUuid) {
         apps.add(addToProgramSummaryListPage(app("pih.app." + programUuid + ".programSummary.dashboard",
                         "pih.app." + programUuid + ".programSummary.dashboard",
                         "fas fa-fw fa-list-alt",
-                        "/coreapps/summarydashboard/summaryDashboard.page?app=" + "pih.app." + programUuid + ".programSummary.dashboard",
```

Comment:
got it... this works, though for clarity, let's just do a global replace in PIH Core of:

"programSummary.dashboard" -> "programEnrollments.dashboard"

and change the "addToProgramSummaryDashboardFirstColumn" to "addToProgramEnrollmentsDashboardFirstColumn"

Reply:
@mogoodrich , there are over 100 matches. are we sure we want to change all those strings?

<img width="1401" alt="Screenshot 2024-09-19 at 10 58 52 AM" src="https://github.com/user-attachments/assets/bde36533-749b-4efa-8888-406d0036facd">

Reply:
Fair enough, let's leave them as-is @cioan